transport/rif_comms/cluster.py

```diff
@@ -0,0 +1,27 @@
+from raiden.tests.integration.network.transport.rif_comms.node import Node, Config
```

Comment:
this module does not belong here, it should reside in `raiden/tests/integration/network/transport/rif_comms`

Reply:
wow 🤦🏻

Reply:
done